substra/cli/interface.py

```diff
@@ -543,29 +546,25 @@ def add_traintuple(ctx, objective_key, algo_key, dataset_key, data_samples, in_m
 @click_option_verbose
 @click.pass_context
 @error_printer
-def add_aggregate_traintuple(ctx, objective_key, algo_key, dataset_key, tag, output_format, config,
-                             profile, user, verbose):
+def add_aggregate_traintuple(ctx, objective_key, algo_key, in_models_keys, worker, rank, tag,
+                             output_format, config, profile, user, verbose):
     """Add aggregate traintuple.
-
-    The option --data-samples-path must point to a valid JSON file with the
-    following schema:
-
-    \b
-    {
-        "keys": list[str],
-    }
-
-    \b
-    Where:
-    - keys: list of data sample keys
     """
     client = get_client(config, profile, user)
     data = {
         'algo_key': algo_key,
         'objective_key': objective_key,
-        'data_manager_key': dataset_key,
     }
 
+    if in_models_keys:
+        data['in_models_keys'] = in_models_keys
+
+    if worker:
+        data['worker'] = worker
+
+    if rank:
```

Comment:
If `rank` is equal to 0, the rank won't be set. In this particular case I think we should check it is not equal to None (that will be the case if rank is not provided).